### ISSUE TITLE
Code injected to core module produces 'redefined' warnings

### ISSUE THREAD

**Issue by jesusbagpuss** [eprints/orcid_support#31]
This block:
https://github.com/eprints/orcid_support/blob/84c916b09af4e28a0af752b2815e944977f08412/cfg/cfg.d/z_orcid_support.pl#L125-L241
when used in a multiple-archive configuration results in:
`Subroutine run_people_with_orcids redefined at (eval 2408) line 171.`

Possibly move this method to a site_lib or lib file.

**Comment by drn05r** [comment]
Hmm, that is a bit worrying as it no suggest their is no isolation between the other archives. When do you see this warning?  When you run `epadmin test` or even when your run `epadmin test ARCHIVEID`?

However, I agree by default this does not really need to be in the archive level configuration.  Most modifications will likely be aesthetic so can be done in CSS.  However, for repository that already augment their creators, editors, etc, in citations (e.g. display institutional creators in bold or link to their browse view) they may need to modify this function, although they are probably best of creating a new function to merge this and their existing augmentations, which would probably avoid the 'redefined' issue.

**Comment by jesusbagpuss** [comment]
Yes - from `bin/epadmin test` and when apache is reloaded, in the error_log.

> no isolation between the other archives
NB Under 3.3.16 this is normal I believe. IIRC you (or someone) added some additional segregation to archives?
